src/content/learn/managing-state.md

```diff
@@ -1006,12 +1006,12 @@ ul, li { margin: 0; padding: 0; }
 
 <LearnMore path="/learn/scaling-up-with-reducer-and-context">
 
-Read **[Scaling Up with Reducer and Context](/learn/scaling-up-with-reducer-and-context)** to learn how state management scales in a growing app.
+Baca **[Mengembangkan Skala dengan *Reducer* dan *Context*](/learn/scaling-up-with-reducer-and-context)** untuk mempelajari bagaimana pengelolaan keadaan (*state*) mengembang pada aplikasi yang berkembang.
```

Comment:
```suggestion
Baca **[Peningkatan Skala dengan *Reducer* dan *Context*](/learn/scaling-up-with-reducer-and-context)** untuk mempelajari bagaimana pengelolaan keadaan (*state*) mengembang pada aplikasi yang berkembang.
```

Please translate links' title according to the [translated title](https://id.react.dev/learn/scaling-up-with-reducer-and-context).